boa/vyper/contract.py

```diff
@@ -611,11 +611,19 @@ def handle_error(self, computation):
             # inspired by answers in https://stackoverflow.com/q/1603940/
             raise strip_internal_frames(b) from None
 
-    def vyper_stack_trace(self, computation):
-        ret = StackTrace([ErrorDetail.from_computation(self, computation)])
-        error_detail = self.find_error_meta(computation.code)
-        if error_detail not in EXTERNAL_CALL_ERRORS:
-            return ret
+    def vyper_stack_trace(self, computation, unknown=False):
+        if unknown:
+            ret = StackTrace(
```

Comment:
i think it still needs to recurse here.

Reply:
it's not a return statement? it will still recurse below as long as there's a child computation